### PR TITLE
controller done

### PR DESCRIPTION
**Pull request by YJouhar47**

### CODE REVIEW

File: config/migrations/202403141548-add-controller-clb.sparql
Comment (lagartoverde) [inline config/migrations/202403141548-add-controller-clb.sparql:1]: Do we need this file? Just curious if it didn't work without it

File: data/db/virtuoso.ini
Comment (lagartoverde) [inline data/db/virtuoso.ini:1]: Don't delete this file on the PR

File: docker-compose.yml
Comment (lagartoverde) [inline docker-compose.yml:1]: You shouldn't change the sync base url of the app

**Review by lagartoverde — CHANGES_REQUESTED** [review]
Looks good overall, some comments on accidental changes you made, and was questioning if that migration is needed for our app

**Comment by lagartoverde** [comment]
Also could you document here which changes are needed to deploy this to qa and prod?

**Comment by lagartoverde** [comment]
Like the changes in the docker-compose.override.yml

File: config/migrations/202403141548-add-controller-clb.sparql
Comment (YJouhar47) [reply config/migrations/202403141548-add-controller-clb.sparql:1]: Yes, otherwise it wont work

**Review by lagartoverde — APPROVED** [review]

**Pull request merged**